doc/classes/String.xml

```diff
@@ -87,26 +90,36 @@
+				[codeblocks]
+				[gdscript]
+				"move_local_x".capitalize()   # Returns "Move Local X"
+				"sceneFile_path".capitalize() # Returns "Scene File Path"
+				[/gdscript]
+				[csharp]
+				"move_local_x".Capitalize();   // Returns "Move Local X"
+				"sceneFile_path".Capitalize(); // Returns "Scene File Path"
+				[/csharp]
+				[/codeblocks]
 			</description>
 		</method>
 		<method name="casecmp_to" qualifiers="const">
 			<return type="int" />
 			<param index="0" name="to" type="String" />
 			<description>
-				Performs a case-sensitive comparison to another string. Returns [code]-1[/code] if less than, [code]1[/code] if greater than, or [code]0[/code] if equal. "less than" or "greater than" are determined by the [url=https://en.wikipedia.org/wiki/List_of_Unicode_characters]Unicode code points[/url] of each string, which roughly matches the alphabetical order.
-				[b]Behavior with different string lengths:[/b] Returns [code]1[/code] if the "base" string is longer than the [param to] string or [code]-1[/code] if the "base" string is shorter than the [param to] string. Keep in mind this length is determined by the number of Unicode codepoints, [i]not[/i] the actual visible characters.
-				[b]Behavior with empty strings:[/b] Returns [code]-1[/code] if the "base" string is empty, [code]1[/code] if the [param to] string is empty or [code]0[/code] if both strings are empty.
-				To get a boolean result from a string comparison, use the [code]==[/code] operator instead. See also [method nocasecmp_to] and [method naturalnocasecmp_to].
+				Performs a case-sensitive comparison to another string. Returns [code]-1[/code] if less than, [code]1[/code] if greater than, or [code]0[/code] if equal. "less than" and "greater than" are determined by the [url=https://en.wikipedia.org/wiki/List_of_Unicode_characters]Unicode code points[/url] of each string, which roughly matches the alphabetical order.
+				With different string lengths, returns [code]1[/code] if this string is longer than the [param to] string, or [code]-1[/code] if shorter. Note that the length of empty strings is [i]always[/i] [code]0[/code].
+				To get a [bool] result from a string comparison, use the [code]==[/code] operator instead. See also [method nocasecmp_to] and [method naturalnocasecmp_to].
 			</description>
 		</method>
 		<method name="chr" qualifiers="static">
 			<return type="String" />
 			<param index="0" name="char" type="int" />
 			<description>
-				Directly converts an decimal integer to a unicode character. Tables of these characters can be found in various locations, for example [url=https://unicodelookup.com/]here.[/url]
+				?? Converts the decimal [param char] into a single Unicode character. You may use [url=https://unicodelookup.com/]unicodelookup.com[/url] as a point of reference.
+				?? Returns a single Unicode character from the decimal [param char]. You may use [url=https://unicodelookup.com/]unicodelookup.com[/url] as a point of reference.
```

Comment:
It's probably better to have a link to official Unicode site - https://www.unicode.org/charts/

Reply:
Which of the two lines is better?

Reply:
Probably second one, since it's not really converting anything, it's more like a type cast.

Reply:
Also, one problem with https://www.unicode.org/charts/ is that it is very difficult to navigate for most common uses.